### DISCUSSION TITLE
Include all Chakra UI stories in another Storybook

### DISCUSSION THREAD

**Discussion by cereallarceny** [chakra-ui/chakra-ui#8888] [General]
### Description

I would like to have the ability to configure the `@chakra-ui/storybook-addon` to add all Chakra UI stories automatically to my local Storybook.

### Problem Statement/Justification

When creating my own Chakra theme, I have to write my own Storybook stories from which to easily preview changes to my components and the theme in general. It would be far better if there were some way to automatically include all the stories included in [Chakra UI's default Storybook](https://chakra-ui.netlify.app/) so that I didn't need to write them myself.

### Proposed Solution or API

I tried using [Storybook composition](https://storybook.js.org/docs/react/sharing/storybook-composition), like such:

```ts
refs: {
  chakra: {
    title: 'Chakra UI',
    url: 'https://chakra-ui.netlify.app/',
  }
}
```

But this wasn't allowed due to cross-origin requests being blocked from localhost. Even if that wasn't the case, however, I doubt these stories would have been injected using my theme. I think the best solution here would be to automatically include the Storybook stories for all components inside package-based libraries like `@chakra-ui/react` so that I can include them like such:

```ts
{
  stories: [
    '../src/**/*.stories.@(js|jsx|ts|tsx)',
    'node_modules/@chakra-ui/react/dist/stories' // Like this
  ],
}
```

**Comment by cereallarceny** [comment]
![Anyone](https://media.giphy.com/media/8FhXc8w45aN32/giphy.gif)

**Comment by jessicalc** [comment]
We, too, have an appetite for this - it would be really helpful to be able to preview our themes applied to the existing Chakra UI stories.

**Comment by jsonhero** [comment]
Would also love to have this please!
